src/routes/GroupSettings/GroupSettingsTab/GroupSettingsTab.js

```diff
@@ -119,6 +127,7 @@ export default class GroupSettingsTab extends Component {
     } = edits
 
     const { locationDisplayPrecision, showSuggestedSkills } = settings
+    const editableMapLocation = group?.locationObject || currentUser.locationObject
```

Comment:
what does this do if there's no location for the group or user? there should be a default location, and/or we could use the browsers get current location functionality